REAMDE

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
1) you have a typo in this file's name: `REAMDE` ;)
2) the same as `LICENSE`: I don't think we need these